matRad_generateStf.m

```diff
@@ -237,20 +237,20 @@
                              ct.resolution, ...
                              stf(i).sourcePoint, ...
                              stf(i).ray(j).targetPoint, ...
-                             {voiTarget});
+                             [{ct.cube{1}} {voiTarget}]);
 
         % find appropriate energies for particles
        if strcmp(stf(i).radiationMode,'protons') || strcmp(stf(i).radiationMode,'carbon')
 
            % target hit
-           if sum(rho{1}) > 0 
+           if sum(rho{2}) > 0 
```

Comment:
I am not 100% sure why this is necessary? Rays are defined such that they "hit" the projection of voxels onto the iso center plane. Or is there the possibility that rays don't hit the target due the ray position padding in line 151?